src/collection-browser.ts

```diff
@@ -984,6 +986,26 @@ export class CollectionBrowser
     const tiles: TileModel[] = [];
     docs?.forEach(doc => {
       if (!doc.identifier) return;
+
+      let loginRequired = false;
+      let contentWarning = false;
+      // Check if item and item in "modifying" collection, setting above flags
+      if (
+        doc.collections_raw?.values &&
```

Comment:
hello, just some clarifying around `doc.collections_raw?.values` - is that an array or a string? line 997 looks like it is looking for an array? should it be `doc.collections_raw?.values?.length`? can/should we use the normalized, not-raw value?

Reply:
[`collections_raw` is] The "natural" collections for an item before augmentation by side-loaded collections like ListsAPI and SimpleLists

The `collection` field above includes things like all of the fav-* collections, whereas this [`collections_raw`] is only the collections that have been directly added in the hierarchy.

Reply:
`doc.collections_raw?.values` is defined as `string[] | undefined` It's either a string array or undefined.
So either: 
- truthy: string array with content
- truthy: empty string array   -- `for .. of` doesn't run
- falsy: undefined

`doc.collections_raw?.values.length` would be `number | undefined`
- truthy: number greater than 0
- falsy: 0
- falsy: undefined

Changed to `.length` as a better-ish guard -- I don't think it matters much, both ways process correctly